packages/kbn-plugin-helpers/src/integration_tests/build.test.ts

```diff
@@ -73,7 +74,8 @@ it('builds a generated plugin into a viable archive', async () => {
   );
 
   expect(filterLogs(buildProc.all)).toMatchInlineSnapshot(`
-    " info deleting the build and target directories
+    "Kibana is currently running with legacy OpenSSL providers enabled! For details and instructions on how to disable see https://www.elastic.co/guide/en/kibana/8.10/production.html#openssl-legacy-provider
```

Comment:
I'll create a follow up PR so we don't have to update these every time we branch.